### PR TITLE
58 periodic cleanup

### LINKED ISSUE #58: Clear ScrapedAuthors collection upon successful operation
Clear ScrapedAuthors collection once authors are added/updated to `Authors` collection successfully. Could be a periodic cleanup.

### PR DESCRIPTION
**Pull request by karelumair**
added periodic clean up and scarping. 

closes #58

### CODE REVIEW

File: constants/app_constants.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
 """Constants"""
 
 QUOTES_URL = "https://quotes.toscrape.com/"
+# Time in minutes
+SCRAPING_SCHEDULER_TIME = 120
+# Time in seconds
+SCRAPED_DATA_CLEAN_UP_TIME = 86400
```
Comment (pratikGhodke1) [inline constants/app_constants.py]:
```suggestion
# Time in minutes
SCRAPING_SCHEDULER_DURATION = 120
# Time in seconds
SCRAPED_DATA_CLEAN_UP_DURATION = 86400
```

Comment (pratikGhodke1) [reply constants/app_constants.py]: Update where these constants are used

File: resources/scrape.py [hunk 2]
```diff
@@ -20,43 +20,16 @@ def get(self):
         current_app.logger.info("GET Scrape Quotes - REQUEST RECEIVED")
```
Comment (pratikGhodke1) [inline resources/scrape.py]: Might have to update this log statement

File: pyproject.toml [hunk 3]
```diff
@@ -25,6 +25,7 @@ redis = "^4.5.1"
 pydantic = "^1.10.4"
 pytest = "^7.2.1"
 pytest-celery = "^0.0.0"
+flask-apscheduler = "^1.12.4"
```
Comment (pratikGhodke1) [inline pyproject.toml:25]: Remove not required dependencies and update poerty.lock file.
You can use - `poetry lock` command to update it.

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
Make the suggested changes

**Comment by karelumair** [comment]
> Make the suggested changes

Done with all required changes

File: pyproject.toml [hunk 4]
```diff
@@ -9,8 +9,6 @@ readme = "README.md"
 python = "^3.10"
 flask = "^2.2.2"
 python-dotenv = "^0.21.0"
```
Comment (pratikGhodke1) [inline pyproject.toml:11]: Are we using dot-env?

Comment (karelumair) [reply pyproject.toml:11]: Yes

**Review by pratikGhodke1 — APPROVED** [review]

**Pull request merged**